sw/tools/create_module/create_mod_qt.py

```diff
@@ -0,0 +1,147 @@
+#!/usr/bin/env python3
+import sys
+from PyQt5 import QtWidgets
+from create_module_ui import Ui_CreateModule_Window
+import components
+import files_create
+import unidecode
+import os
+from lxml import etree
+
+PPRZ_HOME = os.getenv("PAPARAZZI_HOME")
```

Comment:
files in `conf` and `var` are corresponding to `PAPARAZZI_HOME`, but files in `sw` are corresponding to `PAPARAZZI_SRC`.